src/main.rs

```diff
@@ -10,13 +10,14 @@ mod icon;
 // When compiling natively:
 #[cfg(not(target_arch = "wasm32"))]
 fn main() {
+    
     let native_options = eframe::NativeOptions {
         always_on_top: true,
         decorated: true,
         initial_window_size: Some(egui::vec2(550f32, 309f32)),
         resizable: false,
         transparent: true,
-        icon_data: Some(icon::load_icon()),
+        icon_data: Some(icon::load_icon("assets/icon-64x64.ico")),
```

Comment:
`icon::load_icon()` no longer exists as it comes from `icon.rs`

Reply:
Note: Line 8 `mod icon;` should also be removed

Reply:
Just to clarify, you want me to move `load_icon` into main
rs, delete icon.rs, and remove the use of `mod icon`?